regulus/topo/morse.py

```diff
@@ -38,16 +37,16 @@ def msc(data, kind='smale', measure=None, knn=defaults.knn, beta=defaults.beta,
 
     y = data.values.loc[:, measure]
 
-    topo = MorseSmaleComplex(graph=graph, gradient=gradient, max_neighbors=knn, beta=beta,
-                             normalization=norm, aggregator=aggregator, connect=connect)
-    # topopy ver 1.0: remove names
-    # topo.build(X=data.x.values, Y=y.values, names=list(data.x.columns)+[y.name])
+    if 'knn' in graph:
+        beta=0.0
```

Comment:
I should test this, but my thinking is this will essentially make the prune a no-op, this will be an inefficient no-op, but alleviates us from having to write a custom graph. Maybe I can put the logic in nglpy to short-circuit if 0 is given for this parameter.